GRIDS.md

```diff
@@ -6,9 +6,9 @@
 
 ### Australia: AGD66 -> GDA94
 
-*Source*: ICSM - https://github.com/icsm-au/transformation_grids
+*Source*: [ICSM](https://github.com/icsm-au/transformation_grids)
 *Format*: NTv2
-*License*: Creative Commons Attribution 4.0
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
```

Comment:
Please don't mix these link changes for Australia in this PR, the link changes are unrelated to the new grids.

Reply:
I was being lazy. This is of course the correct way to proceed. I'll remove this for now.